### PR TITLE
superTuxKart: 0.9.3 -> 1.0

### PR DESCRIPTION
**Pull request by aanderse**
###### Motivation for this change
http://blog.supertuxkart.net/2019/04/supertuxkart-10-release.html

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by aanderse** [comment]
@peterhoeg how does this look?

**Comment by aanderse** [comment]
@GrahamcOfBorg build superTuxKart

**Review by adisbladis — APPROVED** [review]

File: pkgs/games/super-tux-kart/default.nix [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
+, fribidi, libtool, bluez, libjpeg, libpng, zlib, libX11, libXrandr, enet }:
 
 let
   dir = "stk-code";
 
 in stdenv.mkDerivation rec {
-  name = "supertuxkart-${version}";
-
-  version = "0.9.3";
+  pname = "supertuxkart";
```
Comment (peterhoeg) [inline pkgs/games/super-tux-kart/default.nix]: What happened to ```name```?

Comment (teto) [reply pkgs/games/super-tux-kart/default.nix]: Name is generated from pname/version

**Comment by teto** [comment]
If it's not merged by then, i'll give the binary a try this evening.

File: pkgs/games/super-tux-kart/default.nix (hunk 1 above)
Comment (peterhoeg) [reply pkgs/games/super-tux-kart/default.nix]: Do you have a link to a reference for this? (slightly off-topic for this PR obvoiusly)

Comment (teto) [reply pkgs/games/super-tux-kart/default.nix]: python used to do it and it was extended to stdenv.mkDerivation. You can check in a repl (don't have the reference). There are a few packages in nixpkgs already taking advantages of this. Sry I can't find the PR right now.

**Comment by etu** [comment]
I just built it and tried it both online and offline for like half an hour, seems to work well :smile:

File: pkgs/games/super-tux-kart/default.nix (hunk 1 above)
Comment (peterhoeg) [reply pkgs/games/super-tux-kart/default.nix]: :+1:

**Comment by teto** [comment]
@aanderse thanks for the update. Works perfectly fine on my machine too. Too bad there are so few servers in Asia :(

**Comment by peterhoeg** [comment]
This is getting seriously off-topic but I'll race you on the weekend @teto - Singapore to Tokyo should be fine!

**Comment by teto** [comment]
in order to keep the issue "on-track" what's your IRC nickname (or we could schedule a race via discourse) ? I am more than willing to kick your ass :p

**Comment by peterhoeg** [comment]
@peter:hoeg.com on matrix, peterhoeg on IRC.

For the rest of you guys subscribing to this issue - my apologies for the noise.

**Pull request merged**